src/components/App/App.tsx

```diff
@@ -21,19 +22,23 @@ class App extends React.Component<Props> {
     currentApis: Api[];
     favorites: Api[];
     savedNotes: Notes[]
+    error: string;
   };
   constructor(props: Props) {
     super(props);
     this.state = {
       apiList: [],
       currentApis: [],
       favorites: [],
-      savedNotes: []
+      savedNotes: [],
+      error: ''
     };
   }
 
   componentDidMount() {
-    getApis().then((data) => this.setState({ apiList: data.entries, currentApis: data.entries }));
+    getApis()
+    .then((data) => this.setState({ apiList: data.entries, currentApis: data.entries }))
+    .catch(err => this.setState({ error: err.message }))
```

Comment:
Good call on the catch, had completely forgotten about that until now

Reply:
I like it, nice and clean